clang/lib/CIR/CodeGen/CIRGenCall.cpp

```diff
@@ -889,6 +900,18 @@ RValue CIRGenFunction::emitCall(const CIRGenFunctionInfo &CallInfo,
           auto Results = theCall->getOpResults();
           assert(Results.size() <= 1 && "multiple returns NYI");
           assert(Results[0].getType() == RetCIRTy && "Bitcast support NYI");
+
+          auto region = builder.getBlock()->getParent();
+          if (region != theCall->getParentRegion()) {
+            Address DestPtr = ReturnValue.getValue();
+
+            if (!DestPtr.isValid())
+              DestPtr = CreateMemTemp(RetTy, callLoc, "tmp");
```

Comment:
Let's make this easier to spot: `tmp` -> `tmp.try.call.res` or something like that